### PR TITLE
add docops platform

### PR DESCRIPTION
**Pull request by alearriola-wizeline**
Added pages with information.

### CODE REVIEW

File: tw-docops/tw-mkdocs/docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Installation 
+## List
+
+- Item
+- Item
```
Comment (sAlvaradoM) [inline tw-docops/tw-mkdocs/docs/installation.md:5]: Please make it a bulleted list.

**Review by sAlvaradoM — COMMENTED** [review]
Nice doc! 
I only left one comment

File: tw-docops/tw-mkdocs/docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Overview
+
+## List
+
+- Item
```
Comment (enriquesanchzz2) [inline tw-docops/tw-mkdocs/docs/overview.md:5]: make it a numbered list

File: tw-docops/tw-mkdocs/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Use Markdown to add information to the pages of your site.
+
+For example, here is a list:
+
+1.  Item 1
+2.  Item 2
+3.  Item 3
+
+Here is another list:
```
Comment (enriquesanchzz2) [inline tw-docops/tw-mkdocs/docs/index.md:17]: what kind of list?

**Pull request merged**